docs/howto/set_database_schema.rst

```diff
@@ -0,0 +1,25 @@
+Set the database schema
+-----------------------
+
+Any PostgreSQL database contains one or more *schemas*. Schemas are namespaces really,
+not much more than that. See the `PostgreSQL Schema documentation`_ for more detail on
```

Comment:
This tone is a little bit too harsh. 

> Schemas are Postgres way of implementing namespaces.

Also, you'll need to update the glossary, we now have 2 clashing definitions for what a "schema" is :)

Reply:
Yep, I added a note to the glossary about that. I just mention that the term "schema" is also by Postgres for something else.